engine/samples/hello-cubos/main.cpp

```diff
@@ -0,0 +1,49 @@
+/// [Include]
+#include <cubos/engine/cubos.hpp>
+
+using cubos::engine::Cubos;
+/// [Include]
+
+/// [Hello Cubos]
+void sayHelloCubos()
+{
+    CUBOS_INFO("Hello CUBOS");
+}
+/// [Hello Cubos]
+
+/// [Hello World]
+void sayHello()
+{
+    CUBOS_INFO("Hello");
+}
+
+void sayWorld()
+{
+    CUBOS_INFO("World");
+}
+/// [Hello World]
+
+int main()
+{
+    /// [Engine]
+    Cubos cubos{};
+    /// [Engine]
+
+    /// [Tags]
+    cubos.tag("exampleTagA").before("exampleTagB");
+    cubos.startupTag("exampleTagB");
```

Comment:
Why the second line 🤔? Also, I think renaming the tags to hello and world would be even more clear

Reply:
the second line is there because it was there already :)
sure, i'll rename things.